### ISSUE TITLE
Google Play Protect blocks app installation

### ISSUE THREAD

**Issue by allen820814** [dessant/search-by-image#341]
**System**
* OS name: Android
* OS version: 9
* Browser name: Samsung Internet Browser
* Browser version: 27.0.0.79
* Extension version: 7.1.0

**Bug description**
I recently purchased the "Search by Image for Samsung Internet" app from the Samsung Galaxy Store. However, when I tried to download it, Google Play Protect blocked the installation. The pop-up message stated: "Harmful app blocked. Reason: This app may be harmful."
![Screenshot_20241202-062928_Google Play Store](https://github.com/user-attachments/assets/8e00e31a-9185-4563-b994-066c58c42e53)

**Logs**
Since the installation of the app has been blocked by Google Play Protect, I am unable to provide the browser console output.

**Comment by dessant** [comment]
Hi, thanks for reporting the issue! I have enabled Google Play Protect for testing, and the app seems to be flagged by Google. This is a false positive, the app that wraps the browser extension is supplied and signed by Samsung after we upload the extension code on their platform. I'm going to release a new app version in the next few days, hopefully the new APK will not be flagged without reason by Google.

You can temporarily disable Google Play Protect, but there is also a workaround for installing the app while this issue is present. Tap on the "More details" button in the warning popup, then tap on the "Install anyway" button. You may need to confirm the action with your PIN or fingerprint, and then the app is installed.

<p>
  <img width="380" src="https://github.com/user-attachments/assets/8923040a-3b94-4177-b770-2ef737646c8b">
  <img width="380" src="https://github.com/user-attachments/assets/2dadd850-08e9-4499-828e-630ebbf94298">
</p>

**Comment by allen820814** [comment]
Hello,

Thank you for your prompt response and solution. I will try the method you provided to install the app. Looking forward to the new version release!

Thanks!

**Comment by allen820814** [comment]
Hello,

Thank you for the prompt update! I have successfully updated to version 8.0.1, and now Google Play Protect no longer issues warnings or blocks the app. I really appreciate your help and support!

Thanks!

**Comment by dessant** [comment]
I'm glad it finally works on your device! I've also submitted an appeal [here](https://support.google.com/googleplay/android-developer/answer/2992033?hl=en), but Google appears to have ignored it, which is illegal in the EU, they can't just block an app without a reason. Google also breaks the law again on the Google Play Protect appeal form, stating that developers will not get a response for their submission.

https://support.google.com/googleplay/android-developer/contact/protectappeals

```
All appeal decisions are final and you will not receive a response. Note that we will not provide guidance on the potential compliance of your future implementation, so submissions requesting advice will not receive an answer.
```

The statement above breaks multiple provisions of the [Digital Services Act](https://en.wikipedia.org/wiki/Digital_Services_Act).

To summarize what happened:

I am selling an app on the Galaxy Store, which is the app store of a competitor. The Google Play Protect service built into the Google Play store has flagged the latest version of the app and prevented it from being installed from the Galaxy Store, leading to financial loss for my project. I have tried to appeal the decision, and the appeal was either ignored or rejected. We can't be certain because Google refuses to communicate as part of the appeal process.
